core/templates/dev/head/components/private_info_tooltip/private_info_tooltip_directive.html

```diff
@@ -0,0 +1,37 @@
+<div class="conversation-skin-tutor-card-tooltip">
+      <button class="conversation-skin-tutor-card-tooltip-button">?
+        <span class="conversation-skin-tutor-card-tooltip-text" translate="I18N_TUTOR_CARD_TOOLTIP"></span>
+      </button>
+</div>
+
+<style>
+  .conversation-skin-tutor-card-tooltip {
```

Comment:
Prefix every CSS selector with private-info-tooltip. E.g.:

`private-info-tooltip .conversation-skin-tutor-card-tooltip {...}`

This provides a fairly strong assurance that the CSS is scoped correctly.

Reply:
Done!